README.md

```diff
@@ -1,7 +1,7 @@
 # wallet-address-validator
 Simple wallet address validator for validating Bitcoin and other altcoins addresses in **Node.js and browser**.
 
-Forked from [ryanralph/altcoin-address](https://github.com/ryanralph/altcoin-address).
+Forked from [ognus/wallet-address-validator](https://github.com/ognus/wallet-address-validator).
```

Comment:
can we update the install instructions? i think it will be `npm install @tastyworks/wallet-address-validator` w/ this update